lib/manageiq/session_store.rb

```diff
@@ -0,0 +1,88 @@
+module ManageIQ
+  module SessionStore
```

Comment:
Would `ManageIQ::SessionStoreInterface` be better or too verbose?

Reply:
Or possibly just calling it `ManageIQ::Session` (`Sessions`?), since it really is creating a interface for that as well as keeping track of a `SessionStore`, allowing you to `revoke` a current session(s) (which is a `delete` action for the `@store`).

To me, a `session` is more of a ephemeral concept with some state, `revoke` sort of makes more sense to me instead of delete since you can just request the "state" just be "reinstated" again to get a new one.  Where the `SessionStore` is more of a typical "datastore", and as far as that datastore is concerned, it is just another record to it.

(mostly rationalizing an argument as why a case could be made for two method names, `revoke` v.s. `delete_all` terminology)

Reply:
This was changed to just `ManageIQ::Session`, as mentioned here:

https://github.com/ManageIQ/manageiq/pull/20549#discussion_r489831382